server/src/Utils/addressUtils.ts

```diff
@@ -6,4 +6,13 @@ export const formatToInsertAndGetAddressIdInput = (address: Address) :  InsertAn
     streetValue: address.street || null,
     floorValue: address.floor || null,
     houseNumValue: address.houseNum || null,
-});
\ No newline at end of file
+});
+
+export const formatToNullable = (address : Address): Address => {
+    return {
+        city : address.city !== '' ? address.city : null,
```

Comment:
use the  || syntax

Reply:
but I want to keep the original address in that its not null

Reply:
I mean the original address is not null/undefined , it's ''

Reply:
**UPDATE** : [it works](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1130/commits/dbd2a677252a58d566a126c0b9a8c879a633f5a1) who knew, thanks Javascript!